docs/_clef/Rules.md

```diff
@@ -1,26 +1,64 @@
 ---
 title: Rules
-sort_key: B
+sort_key: C
 ---
 
-The `signer` binary contains a ruleset engine, implemented with [OttoVM](https://github.com/robertkrimen/otto)
+This page provides a fairly low-level explanation for how rules are implemented in 
+Clef. It is a good idea to read the [Introduction to Clef](/docs/clef/introduction)
+and the [Clef tutorial](/docs/clef/tutorial) before diving in to this page.
 
-It enables usecases like the following:
+{:toc}
+-   this will be removed by the toc
+  
+## Introduction
 
-* I want to auto-approve transactions with contract `CasinoDapp`, with up to `0.05 ether` in value to maximum `1 ether` per 24h period
-* I want to auto-approve transaction to contract `EthAlarmClock` with `data`=`0xdeadbeef`, if `value=0`, `gas < 44k` and `gasPrice < 40Gwei`
+Rules in Clef are sets of conditions that determine whether a given action can be
+signed automatically without requiring manual approval from the user. This can be
+useful for automatically approving transactions between a user's own accounts, or
+to approve patterns that are commonly used by applications. This is a UI improvement
+for users making lots of transactions and also a security improvement because transactions
+that do not fit the pattern defined by a ruleset will not be signed unless they are 
+reviewed and manually approved by the end user.
```

Comment:
Rules only govern "approve" vs " disapprove" vs "let someone else decide". The password required for signing is not part of the rules, and the feature to allow clef to remember passwords is independent. 

Also: `and also a security improvement` -- not sure what you mean. Security improvement over what? Over manual review? Hardly, I'd say.

Reply:
Fair points, thanks. I think I meant 'security improvement' compared to having an unlocked account in Geth but on reflection that doesn't make so much sense.

```suggestion
Rules in Clef are sets of conditions that determine whether a given action can be
approved automatically without requiring manual intervention from the user. This can be
useful for automatically approving transactions between a user's own accounts, or
approving patterns that are commonly used by applications. Automatic signing also 
requires Clef to have access to account passwords which is configured independently 
of the ruleset. 
```